olp-cpp-sdk-dataservice-read/src/repositories/PartitionsRepository.cpp

```diff
@@ -272,21 +273,23 @@ CancellationToken PartitionsRepository::GetPartitions(
               }
             }
 
-            auto cachePartitionsResponseCallback = [=](
-                PartitionsResponse response) {
-              if (response.IsSuccessful()) {
-                OLP_SDK_LOG_INFO_F(kLogTag, "put '%s' to cache",
-                                   requestKey.c_str());
-                cache->Put(appendedRequest, response.GetResult(), expiry, true);
-              } else {
-                if (403 == response.GetError().GetHttpStatusCode()) {
-                  OLP_SDK_LOG_INFO_F(kLogTag, "clear '%s' cache",
-                                     requestKey.c_str());
-                  cache->Clear(appendedRequest.GetLayerId());
-                }
-              }
-              callback(response);
-            };
+            auto cachePartitionsResponseCallback =
```

Comment:
```suggestion
            auto cache_partitions_response_callback =
```

Reply:
left intact as only refactoring is not a scope of this PR